src/stories/buttongroup.stories.js

```diff
@@ -8,85 +8,31 @@ export default {
 };
 
 export const Basic = () => html`
-  <cod-button-group data-type="group" data-label="basic example">
+  <cod-button-group data-type="group" label="basic example">
```

Comment:
This shouldn't be changed. The `data-label` attribute on the button group is owned and controlled by the `<cod-button-group>` component which has not been updated yet. This PR should be focused on updating attributes for the button component only.